### PR TITLE
hw3

### PR DESCRIPTION
**Pull request by qnixdev**

### CODE REVIEW

File: src/main/java/com/java_enterprise/home_work_3/Bar.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        } catch (InterruptedException ex) {
+            throw new RuntimeException(ex);
+        }
+
+        System.out.printf("The '%s' in processing order '%s'.%n", bartender, order);
+
+        try {
+            Thread.sleep(200);
+        } catch (InterruptedException ex) {
+            throw new RuntimeException(ex);
```
Comment (kartius) [inline src/main/java/com/java_enterprise/home_work_3/Bar.java:56]: Why did you add two the same blocks of code?

File: src/main/java/com/java_enterprise/home_work_3/Bar.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        }
+
+        this.finishOrder(bartender, order);
+    }
+
+    private void finishOrder(String bartender, String order) {
+        System.out.printf("The '%s' finish creating order '%s'.%n", bartender, order);
```
Comment (kartius) [inline src/main/java/com/java_enterprise/home_work_3/Bar.java:63]: I suppose you need to take order from the queue, isn't it?

File: src/main/java/com/java_enterprise/home_work_3/Bar.java (hunk 1 above)
Comment (qnixdev) [reply src/main/java/com/java_enterprise/home_work_3/Bar.java:56]: to show the "process" of making the drink, so that it is not instantaneous

File: src/main/java/com/java_enterprise/home_work_3/Bar.java (hunk 2 above)
Comment (qnixdev) [reply src/main/java/com/java_enterprise/home_work_3/Bar.java:63]: yes, the order is taken in the proccessOrder() method, this method only takes the name of the order from the previous one
